.github/workflows/diktat_snapshot.yml

```diff
@@ -34,17 +34,11 @@ jobs:
       - name: Build and install
         # install diktat with version 'base_version-pre' to local maven repo
         run: mvn -B clean install -DskipTests=true
-      - name: Revert project version and set proper version for antrun
+      - name: Revert project version and set proper version for diktat check
```

Comment:
how about changing readme also? Adding it as a a separate chapter

Reply:
Yes, will add

Reply:
Added, please take a look